pkg/policy/l4_filter_test.go

```diff
@@ -113,6 +113,52 @@ func (td *testData) addIdentity(id *identity.Identity) {
 	wg.Wait()
 }
 
+// policyMapEquals takes a set of policies and an expected L4PolicyMap. (The assumption is that this policy)
+// selects identity A.
```

Comment:
The second sentence seems weird. I guess `selects identity A` is within the brackets.
```
// policyMapEquals takes a set of policies and an expected L4PolicyMap. 
// The assumption is that this policy selects identity A.
```